incubator/nginx-ldapauth-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "nginx-ldapauth-proxy.fullname" . }}
+  labels:
+    app: {{ template "nginx-ldapauth-proxy.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
```

Comment:
Add `spec.selector`

Reply:
I think my latest commit addresses that.
Is that what you meant?

Reply:
The selector is not correct. You need to fix the `app` label and add the `release` label.